src/mpid/ch4/netmod/ofi/ofi_win.h

```diff
@@ -631,7 +632,7 @@ static inline int MPIDI_NM_mpi_win_lock(int lock_type, int rank, int assert, MPI
 #define FUNCNAME MPIDI_NM_mpi_win_unlock
 #undef FCNAME
 #define FCNAME MPL_QUOTE(FUNCNAME)
-static inline int MPIDI_NM_mpi_win_unlock(int rank, MPIR_Win * win, MPIDI_av_entry_t *addr)
+static inline int MPIDI_NM_mpi_win_unlock(int rank, MPIR_Win * win, MPIDI_av_entry_t * addr)
```

Comment:
I would say the script is wrong then. It doesn't even seem to conform to the [coding standard](https://wiki.mpich.org/mpich/index.php/Coding_Standards#Argument_Declarations). Or perhaps coding standard should be updated?